hat/assets/js/apps/Iaso/domains/dataSources/components/AddNewEmptyVersion.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import { FormattedMessage } from 'react-intl';
+import { LoadingSpinner, useSafeIntl } from 'bluesquare-components';
+import PropTypes from 'prop-types';
+import MESSAGES from '../messages';
+import ConfirmCancelDialogComponent from '../../../components/dialogs/ConfirmCancelDialogComponent';
+import { useCreateSourceVersion } from '../requests';
+import { useFormState } from '../../../hooks/form';
+import InputComponent from '../../../components/forms/InputComponent';
+
+const initialFormState = () => {
+    return {
+        versionDescription: '',
+    };
+};
```

Comment:
Since you return an object without passing a param, you don't need a function here. 

`const initialFormState = {versionDescription:""}`  is enough